.github/workflows/tests.yml

```
@@ -193,42 +196,54 @@ jobs:
           version: 1
 
       - name: Install dependencies
+        env:
+          SUITE_DIRS: ${{ matrix.dirs || matrix.dir || matrix.project }}
```

actionlint (1.7.12)

[error] property "dir" is not defined in object type {dirs: string; procs: number; project: string; shard: number; total_shards: number}

(expression)